mobile/src/notifications/notification.epics.ts

```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ * Copyright (c) Arcadia, Inc. All rights reserved.
+ ******************************************************************************/
+
+import { ActionsObservable, combineEpics, ofType, StateObservable } from 'redux-observable';
+import { AuthActionType, UserLoggedIn } from '../reducers/auth/auth.action';
+import { catchError, ignoreElements, map, switchMap } from 'rxjs/operators';
+import Push from 'appcenter-push';
+import { EMPTY, from, Subject } from 'rxjs';
+import { Action } from 'redux';
+import { openProfile } from '../navigation/navigation.actions';
+import { AppState, DependenciesContainer } from '../reducers/app.reducer';
+import AppCenter from 'appcenter';
+import { installIdReceived, NotificationAction, NotificationActionType } from './notification.actions';
+
+//----------------------------------------------------------------------------
+const notificationsHandler$ = (action$: ActionsObservable<UserLoggedIn>, state$: StateObservable<AppState>) =>
+    action$.ofType(AuthActionType.userLoggedIn).pipe(
+        switchMap(() => {
+            const notification$ = new Subject<Action>();
+
+            Push.setListener({
```

Comment:
Do you do Push.setListener(null) on logout? If not, why?

Reply:
ideally, you'd want to do the same when everybody unsubscribes from your newly created subject, but I assume it works here without any issues because setListener removes all previous listeners implicitly

Reply:
41e46c95ce2cac189c0fc6b08e228d14786c8007